src/components/pages/parameter-or-variable.jsx

```diff
@@ -34,7 +38,7 @@ const ParameterOrVariablePage = React.createClass({
     }
     return (
       <div>
-        <Link className="btn btn-default" to="/">Retour à l'accueil</Link>
+        <Link className="btn btn-default" onClick={this.handleBackButtonClick} to="/">Retour</Link>
```

Comment:
What about simply setting `onClick` to trigger `this.props.history.goBack()`, and changing the `to` to an anchor (`#`), so that you don't have to `preventDefault` and thus to add a wrapper function? Would it work?  🤔